src/ralph/models/xapi/base/actors.py

```diff
@@ -0,0 +1,186 @@
+"""Base xAPI `Actor` definitions."""
+
+from typing import List, Optional, Union
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+from pydantic import AnyUrl, StrictStr, constr
+
+from ..config import BaseModelWithConfig
+from .common import IRI, MailtoEmail
+
+
+class BaseXapiAgentTypeAccount(BaseModelWithConfig):
```

Comment:
Perhaps we could use an Abstract class here ?

Reply:
I am not sure to understand why

Reply:
Yes, I am not sure either; I was probably confused. I believe I was thinking of a class like `AbstractBaseXapiAgent` which would serve as a base for `BaseXapiAgentAccount`, `BaseXapiAgentMbox`, etc.